queries/moonbit/injections.scm

```diff
@@ -0,0 +1,15 @@
+; Copyright 2025 International Digital Economy Academy
```

Comment:
OK, that is excessive. The injection is trivial and boilerplate for all our queries.